source/Willow-Playground-Smalltalks2017/ComponentSummaryStepWebView.class.st

```diff
@@ -37,10 +37,11 @@ ComponentSummaryStepWebView >> initializeCheckboxView [
 
 { #category : #initialization }
 ComponentSummaryStepWebView >> initializeDropDownListView [
-	dropdown := DropDownListWebView applying: [ :theDropdown |  ].
-	
-	dropdown allowAnyOf:
-		#('Smartphone' 'Tablet' 'Destkop' 'Notebook' 'Arduino')
+	dropdown := SingleSelectionWebView
+		dropDownListApplying: [ :theDropdown |  ]
+		applyingToOptions: [  ].
```

Comment:
Either remove the unused argument on the first block, or add arguments to the second block (check that it works before commiting)